Dockerfile

```diff
@@ -4,7 +4,14 @@ ENV NODE_VERSION=8
 
 RUN curl -sL https://deb.nodesource.com/setup_${NODE_VERSION}.x | bash -
 
+# https://github.com/nodejs/help/issues/554#issuecomment-429633801
+RUN printf 'Package: nodejs\n\
+Pin: origin deb.nodesource.com\n\
+Pin-Priority: 1001\n' \
+>> /etc/apt/preferences.d/nodejs
+
 RUN apt-get update \
+  && apt-cache policy nodejs \
```

Comment:
This just prints out information about what package candidates are available to install nodejs from, and which one will be used.